addons/core/translations/resources/en-us.yaml

```diff
@@ -919,6 +920,10 @@ worker:
     type: Type
   tags:
     title_plural: Tags
+    create:
+      title: Create new tags
+      description: Create new api worker tags. You can only add API tags in the admin UI. Config tags are added in the worker config file.
```

Comment:
Is it right to say `you can only add API tags in the admin UI` since you can add it through the CLI too?

Reply:
Good catch. I will update this.